demos/button.scss

```diff
@@ -17,17 +17,18 @@
 @import "./common";
 @import "../packages/mdc-button/mdc-button";
 @import "../packages/mdc-button/mixins";
+@import "../packages/mdc-button/variables";
 @import "../packages/mdc-checkbox/mdc-checkbox";
 @import "../packages/mdc-form-field/mdc-form-field";
 @import "../packages/mdc-typography/mdc-typography";
 
 .mdc-button.secondary-text-button {
   @include mdc-button-ink-color($mdc-theme-secondary);
-  @include mdc-button-ripple((base-color: $mdc-theme-secondary, opacity: .16));
+  @include mdc-button-ripple((base-color: $mdc-theme-secondary, opacity: $mdc-unfilled-button-ripple-opacity));
```

Comment:
Come to think of it, is it possible to use `theme-style` instead of `base-color` for all of these?  This would be `theme-style: secondary` rather than `base-color: $mdc-theme-secondary`, right?

Reply:
Yeah. I kept this one because I thought it might be a good idea to demonstrate both method (with a theme value or with a HEX value), but I don't hold a very strong opinion to that if you insist?

How about if I add a comment here say:
> Besides we would also like to demonstrate the you could use HEX value or variables represent a HEX value to initialize a ripple

How do you feel this way?

Reply:
Since we're using actual theme slots here I'd sooner show us doing it the right way, but if you want we could add a comment above like...

```
// These examples use theme-style to map to a specific theme color.
// You can also specify any hex color via base-color instead of theme-style.
// See MDC Ripple documentation for more information on the ripple config map.
```

Reply:
Addressed this offline. The plan is to make ripple support HEX and theme value agnostically as theme do right now. I will left a todo there to make sure we revisit this parts once we start working on ripple.